mw-webapp/src/logic/wayPage/wayStatistics/StatisticLine.tsx

```diff
@@ -11,6 +11,11 @@ type StatisticLineProps = {
    * Line value (right part)
    */
   value: number | string;
+
+  /**
+   * Line component (right part)
+   */
+  component?: React.ReactNode;
```

Comment:
lets avoid to create additional props. Use polymorphism of description prop: 

```
descritpsion: string | React.ReactNode
```
code will be simplier  in this case